packages/vaadin-login/test/login-overlay.test.js

```diff
@@ -97,6 +97,23 @@ describe('opened overlay', () => {
     pressEnter(vaadinLoginUsername);
     expect(submitSpy.called).to.be.false;
   });
+  it('should focus the username field', () => {
```

Comment:
nit: please add a newline before the test.

Reply:
Done